docs/fundamentals/documentarray/evaluation.md

```diff
@@ -18,7 +18,7 @@ example_da.embeddings = np.random.random([10, 3])
 
 example_da.match(example_da)
 
-example_da.evaluate(metric='precision_at_k')
+example_da.evaluate(metrics='precision_at_k')
```

Comment:
Can we add an example passing multiple metrics?

Reply:
Ok, I will add an example